tests/osgi-tests/src/test/java/org/wso2/carbon/deployment/engine/osgi/utils/OSGiTestUtils.java

```diff
@@ -175,8 +116,8 @@ private static void copyLaunchPropertiesFile() {
      * Copy deployment.yaml file
      */
     private static void copyDeploymentYmlFile() {
-        copy(Paths.get("src", "test", "resources", "conf", "deployment.yml"),
-                Paths.get("conf", "deployment.yml"));
+        copy(Paths.get("src", "test", "resources", "conf", "deployment.yaml"),
```

Comment:
With the new pax exam container, if you want to change any configuration files, you have to use copyfile option, refer,
https://github.com/wso2/carbon-kernel/blob/master/tests/osgi-tests/src/test/java/org/wso2/carbon/osgi/runtime/CarbonRuntimeOSGiTest.java#L119